docs/manuals/deployment/linux/redhat.md

```diff
@@ -0,0 +1,190 @@
+    ```
+    Is provided by Flask because the way of exposing an application on the Internet and
+    developping an application locally is not the same, mainly for security and reliability reasons.
+    This message will disappear when using the web server.
+
+## Prepare the application for deployment
+
+Deploying your application on a remote environment needs a little bit of configuration.
+
+By default, Taipy applications run in Debug mode. Before deploying your application to the Internet,
+you should turn off the Debug mode by setting the _debug_ parameter or the `(Gui.)run()^` to False. <br>
+You must also inform Taipy not to run the application server on its own but rather delegate the execution
+by setting the parameter _run_server_ to False.<br>
+The name of the variable where the Web application is stored is used in the _uWSGI_ configuration:
+this allows the Web server to load the Web application:
+```
+from taipy import Gui
+
+gui_service = Gui(page="# Getting started with *Taipy*")
+web_app = gui_service.run(debug=False, run_server=False)
+```
+In our example, we store this application in the variable _web_app_ (see line 3)
+
+Make sure you upload this code on your targeted machine and install your dependencies with _pip_.
+
+!!! important
+    The entrypoint filename and the app variable name are important for the proper configuration of
+    the _uWSGI_ Web application server. Please, keep them as is or adapt the configuration.
+
+
+## uWSGI application server
+
+To expose your application over the Internet, you must use _uWSGI_ instead of Flask as the application server.
+You would then leverage Nginx to expose the application.
+
+_uWSGI_ can be started manually. But, generally, it's better to start the application automatically when the machine
+starts. To order to do that, you should use [Systemd](https://systemd.io/) which is installed by default on _Ubuntu_.
+
+Beside your application code, run the following command to generate an adapted file for _Systemd_:
+```
+echo """
+[Unit]
+Description=App
+After=syslog.target
+
+[Service]
+ExecStart=uwsgi --http 127.0.0.1:5000 --gevent 1000 --http-websockets --module app:web_app
+WorkingDirectory=`pwd`
+Restart=always
+KillSignal=SIGQUIT
+Type=notify
+StandardError=syslog
+NotifyAccess=all
+User=`whoami`
+
+[Install]
+WantedBy=multi-user.target
+""" > app.uwsgi.service
+```
+Then transfer this file in the correct folder by doing:
+```
+sudo mv app.uwsgi.service /etc/systemd/system/app.uwsgi.service
+```
+
+Now, you can start your application automatically at the startup time of your machine by doing:
+```
+sudo restorecon /etc/systemd/system/app.uwsgi.service
+sudo systemctl daemon-reload
+sudo systemctl start app.uwsgi.service
+sudo systemctl enable app.uwsgi.service
+```
+
+The application is now running locally but is not accessible yet from the Internet.
+
+
+## Exposing to the Internet
+
+To be able to access to your application from the Internet, you should use _Nginx_.
```

Comment:
"To be able to access to" -> "To expose"

Reply:
"To expose your application on the Internet"?